crates/driver/src/domain/competition/mod.rs

```diff
@@ -205,7 +213,13 @@ impl Competition {
 
         let id = settlement.id;
         *self.settlement.lock().unwrap() = Some(settlement);
-        Ok((id, score))
+
+        let reward = solution::Reward {
+            performance_address: self.solver.address(),
+            participation_address: self.solver.address(),
+        };
+        let orders = vec![]; // TODO: implement
```

Comment:
@sunce86 Can you please open an issue and tag it with `colocation` and explain in detail what needs to be done there? Then, reference the issue in this comment as `TODO(#XYZ): calculate the orders` or whatever.

Reply:
I didn't know how to get all ids for all orders in the settlement with the highest score (why settlement has multiple solutions?)